<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="DragArea" name="DropArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Convert_PushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>转化</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetEncoding_ComboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TargetEncoding_Label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;目标编码格式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SourceFileFilter_LineEdit">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SourceEncodingFilter_Label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>135</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;文件后缀过滤&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="SourceEncoding_Label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;源码格式过滤&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_PushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SourceEncodingFilter_LineEdit">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>84</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_PushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragArea</class>
   <extends>QWidget</extends>
   <header location="global">DragArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>TargetEncoding_ComboBox</tabstop>
  <tabstop>SourceEncodingFilter_LineEdit</tabstop>
  <tabstop>SourceFileFilter_LineEdit</tabstop>
  <tabstop>Convert_PushButton</tabstop>
  <tabstop>Clear_PushButton</tabstop>
  <tabstop>Close_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
